resources/console/charts/web/values.yaml

```diff
@@ -36,7 +36,7 @@ replicaCount: 1
 resources: {}
 test:
   acceptance:
-    enabled: false
+    enabled: true
```

Comment:
We could gain more stability with setting this to `false`...

Reply:
It would be the same stability you gain by adding child's training wheels to a motorcycle...